src/utilities/process-reply.js

```diff
@@ -68,7 +68,7 @@ module.exports = async function processReply({
         if (supportedLanguage) {
           const formattedCode = formatter(content, supportedLanguage);
           ref.channel.send(
-            formatMessageWithCodeblock(languageGuesses[2], formattedCode)
+            formatMessageWithCodeblock(supportedLanguage, formattedCode)
```

Comment:
Isn't this also impossible?

Reply:
This was the third option when `language-classifier` was used....yes this code is unreachable... I'll look into it in more details after a few hours...

Reply:
I've removed the third `case` in the `switch` statement as it is unreachable after removing `language-classifier`.